sdk/keyvault/azcertificates/go.mod

```diff
@@ -1,6 +1,10 @@
 module github.com/Azure/azure-sdk-for-go/sdk/keyvault/azcertificates
 
-go 1.17
+go 1.18
+
+replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../../azcore
+
+replace github.com/Azure/azure-sdk-for-go/sdk/keyvault/internal => ../internal
```

Comment:
Might be better to remove the `replace` directives and reference the versions by commit hash instead.  This would allow one to reference `azcertificates` by commit hash as well and things will just work.